site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -178,23 +187,44 @@ class RegisterUser extends Controller
     return redirect('/');
   }
 }
+//..
 ```
 
-_Note: FusionAuth includes several other options for user registration. Be sure to [check out the documentation](https://fusionauth.io/docs/v1/tech/apis/registrations) to make sure you include all the appropriate options._
+Finally, you'll need to create two routes to register new users: one to display the registration form and another to process the data and interact with the FusionAuth client. Add the following lines to your `routes/web.php` file (remove the welcome route):
```

Comment:
Curious why you moved the routes down. Did that just make more sense?

Reply:
If the routes are out of order you will get an error....  
You need to run 

```shell
php artisan make:controller RegisterUser --invokable
```
before you add the route in `web.php`

I will another `important` tip!